modules/time/time.js

```diff
@@ -0,0 +1,7 @@
+import { timebox } from '../selector/selectors.js';
+import { DateTime } from '../luxon/src/luxon.js';
+
+export function time() {
```

Comment:
- [ ] Kindly refactor this function to use ES6 syntax as it makes your code look neater and takes less space.